index.html

```diff
@@ -17,205 +17,228 @@
-              <li>Bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. h
-            as been the industry's standard</p>
-          
-          <hrx class="tags">
-            <ul>
-              <li>HTML</li>
-              <li>Bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. 
-            has been the industry's standard</p>
-          
-          <hrx class="tags">
-            <ul>
-              <li>HTML</li>
-              <li>Bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. 
-            has been the industry's standard</p>
-          
-          <hrx class="tags">
-            <ul>
-              <li>HTML</li>
-              <li>Bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. 
-            has been the industry's standard</p>
-          
-          <hrx class="tags">
-            <ul>
-              <li>HTML</li>
-              <li>Bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </hrx>    
-          <button type="button">See Project</button>
-        </hrx>
-        <hrx class="work-cards ">
-          <h2>Profesional Art Printing Data</h2>
-          <p>A daily selection of privately personalized reads; 
-            no accounts or sign-ups required. 
-            has been the industry's standard</p>
-          
-          <hrx class="tags">
+    
+      <section class="about-us">
+        <h2 class="primary-text">About me</h2>
+        <p class="description">Hello I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+          <button type="button">Get My Resume</button>
+      </section>   
+  
```

Comment:
- [ ] Kinldy make sure you use the correct typo (`font-familly, size`) as the provided design 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199579333-e1f9e82d-d396-4fe0-b377-4999ef28c443.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199579446-fea7ca38-c43b-48b9-b610-9e667cc1e7b3.png)